### ISSUE TITLE
Crashes when using latest 2.0 release for 1.16

### ISSUE THREAD

**Issue by Epikest** [stashymane/extra-sounds#54]
Minecraft: 1.16_combat-6
Mod: 2.0.1+1.16

## Details
The game crashes when I use the latest 2.0 release but does not crash when I use a 1.0 release

[Please run the game with `-Dextrasounds.debug=true` added to your Java arguments and attach a log file as well.](https://paste.ee/p/k6qgu)

**Comment by Epikest** [comment]
@stashymane why was this closed? it isn't resolved yet

**Comment by stashymane** [comment]
May have closed this when mass-closing the 1.19 issues, my bad

**Comment by stashymane** [comment]
Yes, I may have compiled with Java 16, but there are a few other issues blocking an update, the main one being a severe lack of time. I'll try my best to work on the mod in a week or two.
